src/Refinery/ProofState.hs

```diff
@@ -1,21 +1,23 @@
-{-# LANGUAGE ViewPatterns #-}
+{-# LANGUAGE LambdaCase #-}
```

Comment:
nit: formatting for these languages is all crazy

Reply:
Yeah HLS likes to do weird stuff sometimes